_posts/2018-09-29-cfp-in-progress.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Call for Proposals in progress, keynotes announcement"
+authors:
+  - skade
+  - apiraino
+
+---
+
+# Call for Proposals
+
+Last week we closed the CfP (Call for Proposals) submission window, we allowed a couple more days to really give everyone a chance to submit.
+
+We received 50 interesting proposals to fill up the talks slots for the upcoming Rustfest in Rome, thank you so much! Now it's our turn to open the envelopes and choose those that will make to the conference. As usual, the choice will be tough, all the topics submitted looks absolutely .
+
+Like every year, there's a preliminary selection; here is a draft of the schedule, to be refined and finalized:
+
+| Talk type     | amount planned |
+| ------------- | --------------:|
+| Keynotes      | 2              |
+| 15min format  | 2              |
+| 30min format  | 7              |
+| 45min format  | 2              |
+
+# Keynotes
+
+We are excited to announce the line-up of the speakers for the keynotes:
+
+### Aaron Turon
+
+He works at Mozilla on the Rust programming language, together with an incredible, worldwide community of contributors.
```

Comment:
I'll write you one for that.

Aaron Turon is the current lead of the Rust project, building the Rust language and ecosystems together with an incredible, worldwide community of contributors.

Aaron has an academic background in concurrency as well as verification and programming language design.

He currently lives in Portland, Oregon.